resources/views/dashboard/training/edit_ticket.blade.php

```diff
@@ -115,13 +115,13 @@
             <div class="col-sm-6">
                 <div class="form-group">
                     <label for="comments" class="form-label">Comments (Visible to Controller and other Trainers)</label>
-                    {{ html()->textarea('comments', $ticket->comments)->class(['form-control']) }}
+                    {{ html()->textarea('comments', null)->class(['form-control']) }}
                 </div>
             </div>
             <div class="col-sm-6">
                 <div class="form-group">
                     <label for="trainer_comments" class="form-label">Trainer Comments (Visible to Only Other Trainers)</label>
-                    {{ html()->textarea('trainer_comments', $ticket->ins_comments)->class(['form-control']) }}
+                    {{ html()->textarea('trainer_comments', null)->class(['form-control']) }}
```

Comment:
Same as above. The second parameter in textarea is the value. In this case, we want to retain the $ticket->ins_comments variable because that will fill the form field with the existing value.